### PR TITLE
Fix/navigation

### PR DESCRIPTION
**Pull request by Romainhir**
Fix bug with navigation and the main activity

### CODE REVIEW

File: app/src/main/java/com/android/unio/MainActivity.kt [hunk 1]
```diff
@@ -33,11 +32,18 @@ fun UnioApp() {
...
+    //      navigation(startDestination = Screen.HOME, route = Route.HOME) {
+    //        composable(Screen.HOME) { HomeScreen(navigationActions) }
+    //      }
+    //      navigation(startDestination = Screen.EXPLORE, route = Route.EXPLORE) {
+    //        composable(Screen.EXPLORE) { ExploreScreen(navigationActions) }
+    //      }
+    //  }
+    navigation(startDestination = Screen.HOME, route = Route.HOME) {
+      composable(Screen.HOME) { HomeScreen(navigationActions) }
+
```
Comment (AlouchLaBouche) [inline app/src/main/java/com/android/unio/MainActivity.kt:46]: Implement here the different screens `Account` and `Saved`

**Review by AlouchLaBouche — CHANGES_REQUESTED** [review]
The app crashes when we try to navigate to the `Saved` and `Account` screens. This is because they are not implemented in the `NavHost`.
Futhermore, the `Map` button should not be in the `BottomNavigationMenu` But should be a `Floating Action Button`. (refer to the [Figma](https://www.figma.com/design/rlvqRyIsbdSyEbXu8Xp7yG/Unio?node-id=0-1&node-type=canvas&t=LvJdEkDqP8Gs3sbG-0))

File: app/src/main/java/com/android/unio/MainActivity.kt (hunk 1 above)
Comment (oskar-codes) [reply app/src/main/java/com/android/unio/MainActivity.kt]: I just worked on this and it should be fixed by now.

**Review by oskar-codes — APPROVED** [review]

**Review by oskar-codes — APPROVED** [review]
The fix is working properly, the unit/connected tests pass locally and on the CI, and the application runs correctly on its own.

**Review by AlouchLaBouche — APPROVED** [review]

**Review by leagrieder — COMMENTED** [review]
### PR Review

Good fix to improve the navigation system, well done, this is a good PR

**Pull request merged**